.github/ISSUE_TEMPLATE/aws-access-request.yml

```diff
@@ -0,0 +1,90 @@
+        - Crystal Moultrie
+        - Doris Lin
+        - Vilay Senthep
+        - Other
+    validations:
+      required: true
+  - type: dropdown
+    id: vendor
+    attributes:
+      label: Vendor Onboarding Representative Name
+      description: This is the person from the contracting company responsible for your onboarding process.
+      options:
+        - Oddball - Amber Malcolm
+        - Insignia - Kimberly S. Cole
+        - Agile Six - Anthony M. Arashiro
+        - GovernmentCIO - Kimberly O. West
+        - Magnum Opus - Paula G. Mendoza
+        - Liberty IT -  Michael G. Lovejoy
+        - Accenture Federal Services - Alexa Elliot
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: requestor-name
+    attributes:
+      label: Your Name
+      description: Please provide your name
+      placeholder: e.g. Jane Doe
+    validations:
+      required: true
+  - type: input
+    id: email
+    attributes:
+      label: Your Email
+      description: Please provide your work email
+      placeholder: e.g. jane.doe@company.com
+    validations:
+      required: true
+  - type: input
+    id: team-name
+    attributes:
+      label: Your Team, Role, and Company
+      description: Please provide the name of your team, your role on that team, and the name of the company you work for
+      placeholder: e.g. Platform Security, Backend Engineer, Ad Hoc
+    validations:
+      required: true
+  - type: textarea
+    id: aws-access
+    attributes:
+      label: Desired AWS Access
+      description: Please list which resources (bucket names, parameter store paths, EC2 instance name, etc.) would you like to access. Also list what type of access will you need (read, read-write, etc.). If there is a teammate whose permissions you'd like to copy, please list their name.
+      placeholder: I need access to S3 to read uploaded files from the /tools-team/uploads bucket.
+  - type: input
+    id: expiration-date
+    attributes:
+      label: Access Expiration
+      description: The requested access will be revoked at the start of this date. Enter a date (and time if applicable) or 'ongoing' for non-expiring access.
+      placeholder: timestamp
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        For SOCKS access, open a [SOCKS Access Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=external-request%2Coperations%2Cops-access-request&template=socks-access-request.yml&title=Access+for+%5Bindividual%5D)
+        If you need additional infrastructure, please open a separate [Ops request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=operations%2C+devops%2C+needs-grooming&template=ops_issue_template.md&title=) to create that infrastructure.
+
+CC: @department-of-veterans-affairs/vsp-operations , @department-of-veterans-affairs/operations-aws-approvers.
```

Comment:
@jhouse-solvd You mentioned that it would be nice to notify the infrastructure team and AWS approvers automatically. Is this what you mean by that?